data/sql/updates/pending_db_world/arena-rewards.sql

```diff
@@ -0,0 +1,72 @@
+DELETE FROM `arena_season_reward_group` WHERE `arena_season` IN (1, 2, 3, 4);
+INSERT INTO `arena_season_reward_group` (`id`, `arena_season`, `criteria_type`, `min_criteria`, `max_criteria`, `reward_mail_template_id`, `reward_mail_subject`, `reward_mail_body`, `gold_reward`) VALUES
+-- Season 4 (Brutal)
+(21, 4, 'abs', 1, 1, 0, '', '', 0),
+(22, 4, 'pct', 0, 0.5, 262, '', '', 0),
+(23, 4, 'pct', 0.5, 3, 0, '', '', 0),
+(24, 4, 'pct', 3, 10, 0, '', '', 0),
+(25, 4, 'pct', 10, 35, 0, '', '', 0),
+-- Season 3 (Vengeful)
+(26, 3, 'abs', 1, 1, 0, '', '', 0),
+(27, 3, 'pct', 0, 0.5, 211, '', '', 0),
+(28, 3, 'pct', 0.5, 3, 0, '', '', 0),
+(29, 3, 'pct', 3, 10, 0, '', '', 0),
+(30, 3, 'pct', 10, 35, 0, '', '', 0),
+-- Season 2 (Merciless)
+(31, 2, 'abs', 1, 1, 0, '', '', 0),
+(32, 2, 'pct', 0, 0.5, 182, '', '', 0), -- Mail needs verification
```

Comment:
Mail template 182 doesn't seem right to me, not that I care much. But as an alternative, we could use `reward_mail_subject` and `reward_mail_body` instead of the template.

Reply:
That's a good call, at least it would show up differently in the DB to distinguish it from the likely good mail template usages.